### DISCUSSION TITLE
SRM Notes for 25-07-2024

### DISCUSSION THREAD

**Discussion by glopesdev** [harp-tech/protocol#102] [SRM Notes]
### 
- 🚀Small Issue (10min)
- 🚅Medium Issue (20min)
- 🐢Large Issue (30min) 

### Benchmarks
- New RP2040 clock synchronizer with extra functionality 🚅

### Device Updates
- Publish packages from devices with regenerated interfaces 🚀

**Comment by artursilva0** [comment]
@glopesdev These PRs interfaces were already tested and are being used offline. It would be nice to have them released whenever is possible:
https://github.com/harp-tech/device.multipwm/pull/1
https://github.com/harp-tech/device.rgbarray/pull/1
https://github.com/harp-tech/device.cameracontroller/pull/1

This one had an update for a bug fix:
https://github.com/harp-tech/device.syringepump
